frontend/src/routes.js

```diff
@@ -11,39 +10,36 @@ const app = express()
 app.set('view engine', 'njk');
 app.use(express.static(path.join(__dirname, "public")));
 
-nunjucks.configure('viewdir', { 
+nunjucks.configure('viewdir', {
     express: app 
 }); 
 
-const api_url = process.env.API_URL;
-
 // Add routes
 app.get('/', async (req, res) => {
-    const response = await fetch(api_url+'api/',{method:'GET',headers:{}})
-    const data = await response.json();
-    res.render('homePage', {test: data}); 
-
+    res.render('homePage'); 
   });
 
+const jobs = require('./jobRoles')
 app.get('/job-roles', async (req, res) => {
-    const response = await fetch(api_url+'job-roles',{method:'GET',headers:{}})
-    const roleData = await response.json();
-    res.render('jobRolesPage', {items: roleData}); 
-
+    res.render('jobRolesPage', {items: await jobs.getJobRoles()}); 
```

Comment:
```suggestion
    const roleData = await jobs.getJobRoles();
    res.render('jobRolesPage', {items: roleData}); 
```
Seems like theres a mixture of storing it in a variable and calling it straight in the render, would probably stick to storing it in a variable to keep all the requests the same